src/plugins/vis_types/timeseries/public/trigger_action/index.test.ts

```diff
@@ -87,7 +87,7 @@ describe('triggerTSVBtoLensConfiguration', () => {
           ...model.series[0],
           metrics: [
             {
-              type: 'percentile_rank',
+              type: 'std_deviation',
```

Comment:
ℹ️ This test is testing the scenario that a metric agg is not supported in Lens. As we support now  the `percentile_rank` I changed it to `std_deviation` that is not supported